stetho/src/main/java/com/facebook/stetho/inspector/database/SQLiteDatabaseCompat.java

```diff
@@ -31,9 +32,10 @@
   private static final SQLiteDatabaseCompat sInstance;
 
   static {
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
All the changes in this file are due to the fact that SQLCipher doesn't support the later APIs, forced the older API and replaced another API call with a `PRAGMA`